### ISSUE TITLE
About getting comment content in onValueChanged()

### ISSUE THREAD

**Issue by fengzhihenxs** [surveyjs/survey-library#2215]
Description: The project needs to get the comment content when modifying the comment. The surveyjs version used before was 1.0.48, and the comment content can be obtained in onValueChanged(). Now upgrade to the latest version 1.7.10, modify comments will not enter onValueChanged () method.
![image](https://user-images.githubusercontent.com/37610470/84219434-1bed2780-ab03-11ea-8801-56dea2158616.png)

Question: In the new version 1.7.10, how to judge whether the comment of a question is modified, and when the revision is made, to obtain the content of the comment?

**Comment by andrewtelnov** [comment]
@fengzhihenxs Could you please modify [the example](https://plnkr.co/edit/Hzqdsa65OIA0iVAQ) to show the issue?

Thank you,
Andrew

**Comment by fengzhihenxs** [comment]
First of all thank you for your reply.
Wait, I create a demo

**Comment by fengzhihenxs** [comment]
@andrewtelnov    This is a simple demo, using angular9, one of the requirements is to get the changed question including comments.
[svrvey.zip](https://github.com/surveyjs/survey-library/files/4757151/svrvey.zip)

Your jQuery version of the example is possible, but the angular version does not seem to work

**Comment by andrewtelnov** [comment]
@fengzhihenxs Is it possible to modify my example? Based on my experience, we will spend about an hour to run/debug your project and then we will find that something happens in your code. The problem can be in our code, but it is less likely.

You can also debug your app and set the break-point this [this line](https://github.com/surveyjs/survey-library/blob/master/src/survey.ts#L4421). Probably it will be the fastest way to find the issue.

Thank you,
Andrew

**Comment by fengzhihenxs** [comment]
@andrewtelnov Thanks, it should be the reason for my code, I have solved it.

By the way, if there are multiple pages, is there a way to sort the questions on each page starting from 1?

**Comment by andrewtelnov** [comment]
@fengzhihenxs Please set [survey.showQuestionNumbers](https://surveyjs.io/Documentation/Library?id=surveymodel#showQuestionNumbers) property to "onpage".

Thank you,
Andrew

**Comment by devinv44** [comment]
I am having a similar issue trying to use this in React. I have created a survey with a radiogroup, a rating bar, and some comment / text fields. At first, I put the survey in my Index.js file and everything worked perfectly. However, I actually needed the survey in one of my child components. Once I put it there all of the data stopped persisting. I used the components State and "onValueChanged" to persist the data which fixed the radio group and rating bar, however, I noticed that with the comment / text fields "onValueChanged" was never called and thus my state was never updated.

Is it possible to have the survey on a child component, and if so is there something specific I need to do to get it working properly?
